src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/scenarios/EXUIIACMC.scala

```diff
@@ -88,23 +128,38 @@ object EXUIIACMC {
     .group("XUI${service}_060_StartAppealChecklist") {
       exec(http("XUI${service}_060_StartAppealChecklist")
       .post("/data/case-types/Asylum/validate?pageId=startAppealchecklist")
-      .headers(IACHeader.headers_9)
-      .body(StringBody("{\n  \"data\": {\n    \"checklist\": {\n      \"checklist5\": [\n        \"isResidingInUK\"\n      ],\n      \"checklist2\": [\n        \"isNotDetained\"\n      ],\n      \"checklist7\": [\n        \"isNotEUDecision\"\n      ]\n    }\n  },\n  \"event\": {\n    \"id\": \"startAppeal\",\n    \"summary\": \"\",\n    \"description\": \"\"\n  },\n  \"event_token\": \"${event_token}\",\n  \"ignore_warning\": false,\n  \"event_data\": {\n    \"checklist\": {\n      \"checklist5\": [\n        \"isResidingInUK\"\n      ],\n      \"checklist2\": [\n        \"isNotDetained\"\n      ],\n      \"checklist7\": [\n        \"isNotEUDecision\"\n      ]\n    }\n  }\n}"))
+      .headers(Headers.commonHeader)
+      .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-data-validate.v2+json;charset=UTF-8")
+      .header("x-xsrf-token", "${XSRFToken}")
+      .body(ElFileBody("bodies/iac/IACStartChecklist.json"))
       .check(status.is(200)))
```

Comment:
All of these status checks for HTTP 200/304 status codes are redundant, as it's checked by default by Gatling. See https://gatling.io/docs/gatling/reference/current/http/check/#http-status